flexbox/flexbox-styles.css

```diff
@@ -0,0 +1,182 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.container {
+    margin: 10px;
+    padding: 10px;
+    display: grid;
+    grid-template-areas: 
+    "header header header"
+    "leftsidebar main rightsidebar"
+    "leftsidebar footer footer";
+    grid-template-columns: 100px 1fr 150px;
+    grid-template-rows: 40px 1fr 40px;
+    gap: 10px;
+    background-color: rgb(27, 80, 129);
+}
+
+.container > div {
+    background-color: rgb(157, 186, 197);
+}
+
+.main .cat-1{grid-area: cat-1}
+.main .cat-2{grid-area: cat-2}
+.main .cat-3{grid-area: cat-3}
+.main .cat-4{grid-area: cat-4}
+.main .cat-5{grid-area: cat-5}
+.main .cat-6{grid-area: cat-6}
+.main .cat-7{grid-area: cat-7}
+.main .cat-8{grid-area: cat-8}
+
+.main {
+    grid-area: main;
+    padding: 10px;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 200px 200px 200px 200px;
+    gap: 10px;
+    grid-template-areas:
+    'cat-1 cat-1 cat-2'
+    'cat-1 cat-1 cat-5'
+    'cat-6 cat-7 cat-5'
+    'cat-3 cat-4 cat-8';
+}
+
+.main-img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+}
+
+.main-item {
+    position: relative;
+    overflow: hidden;
+}
+
+.meow {
+    position: absolute;
+    top: 100%;
+    left: 100%;
+    transform: translate(-100%, -100%);
+    padding: 10px 10px 10px 100%;
+    font-size: 25px;
+    font-weight: bold;
+    color: white;
+    background-color: rgba(0, 0, 0, 0.5);
+    opacity: 0;
+    transition: opacity 0.7s;
+}
+
+.main-item:hover .meow {
+    opacity: 1;
+}
+
+.header {
+    grid-area: header;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.nav-bar {
+    list-style-type: none;
+    padding: 0px;
+}
+
+.nav-bar li {
+    display: inline-block;
+    padding: 10px;
+}
+
+.nav-bar li a {
+    color: rgb(27, 80, 129);
+    font-size: 20px;
+}
+
+.nav-bar li a:hover {
+    color: red;
+}
+
+.left-sidebar {
+    grid-area: leftsidebar;
+}
+
+.left-sidebar ul {
+    margin: 5px;
+    float: left;
+    display: block;
+    list-style-type: none;
+}
+
+.left-sidebar ul li {
+    padding: 5px;
+}
+
+.left-sidebar ul li a {
+    text-decoration: none;
+    color: rgb(27, 80, 129);
+    font-size: 15px;
+}
+
+.left-sidebar ul li a:hover {
+    color: red;
+}
+
+.right-sidebar {
+    grid-area: rightsidebar;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+}
+
+.right-sidebar span {
+    margin-bottom: 5px;
+    transition: opacity 1s;
+}
+
+.right-sidebar-img {
+    position: relative;
+}
+
+.right-sidebar-img img {
+    width: 100%;
+    height: auto;
+    object-fit: cover;
+    transition: opacity 0.7s;
+}
+
+.woff {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
```

Comment:
Same here. 
Just make it full width, full height and positioned top:0 and left:0
That's should be it